assets/javascript/game.js

```diff
@@ -0,0 +1,260 @@
+    billboardDiv.style.width = "400px";
+    billboardDiv.style.height = "100px";
+    billboardDiv.style.border = "thin black solid";
+    billboardDiv.style.visibility = "hidden";
+    billboardDiv.style.position = "fixed";
+    billboardDiv.style.left = "420px";
+    billboardDiv.style.top = "250px";
+    billboardDiv.style.marginleft = "-200px";
+    billboardDiv.style.marginRight = "34px";
+    billboardDiv.style.paddingTop = "24px";
+    billboardDiv.style.textAlign = "center";
+    billboardDiv.style.background = "crimson";
+    billboardDiv.style.fontsize = "24px";
+    billboardDiv.style.color = "whitesmoke";
+    
+
+    document.getElementById("gamecontainer").appendChild(billboardDiv);
+
+
+}
+function updateBillboardContainer(theText) {
+    var extraText = "<br><br>Hit ENTER to continue";
+    if(endgame == true){
+        extraText = "<br><br>Hit ENTER to restart game. ";
+
+ }
+ document.getElementById('billboardContainer').innerHTML = theText + extraText;
+
+ document.removeEventListener("keydown", gameTurnLoop);
+
+ document.addEventListener("keydown", billboardListener, false);
+
+ document.getElementById('billboardContainer').style.visibility = "visible";
+
+}
+
+function billboardListener(e){
+    var keyCode = e.keyCode;
+
+    if(keyCode==13) {
+        document.removeEventListener("keydown", billboardListener);
+
+        document.getElementById('billboardContainer').style.visibility = "hidden";
+
+        if(endgame == true) {
+            resetGame();
+        } else if(restartLevel == true) {
+
+            currentQuestion++;
+
+            if(currentQuestion >= answers.length){
+                endgame = true;
+                updateBillboardContainer("UNLIMITED POWER!!");
+                
+
+
+            }
+            else {
+                restartLevel = false;
+                doLevel();
+            } 
+        }   
+
+
+        else {
+            document.addEventListener("keydown", gameTurnLoop, false);
+        }
+    }
+
+}
+
+
+function gameTurnLoop (e) {
+    
+    var keyCode = e.keyCode;
+    var currentGuess = String.fromCharCode(e.keyCode).toLowerCase();
+    
+    if (e.keyCode >= 65 && e.keyCode <= 90) {
+        var currentLetter = 0; 
+        var guessedCorrectly = false;
+        
+        if(guessed.indexOf(currentGuess) != -1){
+            updateBillboardContainer("you guessed " + currentGuess + " already!");
+        } else {
+            guessed.push(currentGuess);
+
+            answers[currentQuestion].split('').forEach(function(curLetter){
+                if(curLetter.toLowerCase() == currentGuess){
+                    guessedCorrectly = true;
+                    correctGuesses++;
+                    updateBillboardContainer("I can feel the power flowing through you. Correct!!");
+
+                    var theID = "letter" + currentLetter;
+                    document.getElementById(theID).innerHTML = currentGuess;
+
+                }
+
+                currentLetter++;
```

Comment:
work on fixing indentation and removing extra blank lines.  After removing blank lines you can right click and say format document to automatically fix indentation